Jenkinsfile

```diff
@@ -1,22 +1,22 @@
 pipeline {
+    environment {
+        ZENDESK_JAVA_CLIENT_TEST_URL             = credentials('ZENDESK_JAVA_CLIENT_TEST_URL')
+        ZENDESK_JAVA_CLIENT_TEST_USERNAME        = credentials('ZENDESK_JAVA_CLIENT_TEST_USERNAME')
+        ZENDESK_JAVA_CLIENT_TEST_PASSWORD        = credentials('ZENDESK_JAVA_CLIENT_TEST_PASSWORD')
```

Comment:
probably not all of these should be credentials... and the username/password should be the same credentials not two separate SecretText credentials

Reply:
For now I created distinct entries to have the build working (feel free to edit
https://opensource.ci.cloudbees.com )

https://jenkins.io/doc/pipeline/tour/environment/#credentials-in-the-environment

I'm not sure we can control the naming of the generated environment variables if we use a Username/Password credential

Reply:
https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/e2f89933976a8a2885ca5b061ab3fb1d8a8fe7bb/pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/model/CredentialsBindingHandler.java#L162-L165

A usernamepassword credential will be bound to three env vars:

`FOO` which gets username:password
`FOO_USR` which gets username
`FOO_PWD` which gets password

I would like some guidance from @abayer on how this code gets used though

Reply:
yep @stephenc this is what I understood. We just need to rename our properties to match this pattern usr/pwd instead of username/password
Not a problem from my POV